torchreid/utils/torchtools.py

```diff
@@ -307,6 +318,20 @@ def load_pretrained_weights(model, file_path='', pretrained_dict=None):
             'please check the key names manually '
             '(** ignored and continue **)'.format(message)
```

Comment:
since now we raise an error, the comment `** ignored and continue **` is not needed anymore.

Reply:
Ok, I will remove it

Reply:
Removed